wazo_dird/plugins/graphql/schema.py

```diff
@@ -12,11 +15,13 @@
     relay,
 )
 
+if TYPE_CHECKING:
+    from wazo_dird.plugins.source_result import _SourceResult
+    from .resolver import ResolveInfo
 
-class ContactInterface(Interface):
-    class Meta:
-        interfaces = [relay.Node]
 
+class ContactInterface(Interface):
+    # These are two words... should be first_name and last_name
```

Comment:
Don't understand the relevance of this comment

Reply:
Sorry, it was just a kind of a reminder that we should perhaps correct this mistake at some point. It's everywhere.